charts/kubermatic/templates/default-cluster-overrides-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-cluster-overrides
+data:
+  yaml: |
```

Comment:
THis is an unfortunate naming, because it makes it hard to tell `data.yaml` from a filename and a path expression. Can we make the key just `config.yaml` (i.e. the path being `spec."config.yaml"`)?